app/routes/docs_.schema.$version.$/route.tsx

```diff
@@ -112,7 +85,28 @@ export default function () {
             examples={examples}
           />
         ) : (
-          <SchemaDefinition />
+          <>
+            <SchemaDefinition />
+            <CardGroup>
+              {...data.map((x: GitContentDataResponse) => (
```

Comment:
Change `useLoaderData()` to `useLoaderData<typeof loader>()` so that this type is automatically inferred.

Reply:
This change makes the following message appear:
Parameter 'x' implicitly has an 'any' type.

Reply:
What is it inferring the type of `data` is?

Reply:
It is inferring the type as `any`, could that be because the loader function returns JSON?

Reply:
No, I don't think so. You declared it like this in the loader:

```
let data: GitContentDataResponse[]
```

Reply:
```
return json(
    { data, jsonContent, examples, versions },
    { headers: publicStaticShortTermCacheControlHeaders }
  )
```
seems to break the ties of the typing in the loader, as the returned object from the loader is a `Response`:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/549e2f55-4beb-4fe2-b295-b3390fa9d87b)


 where as 

```
return{ data, jsonContent, examples, versions }
```
keeps the types:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/ad8ca174-2bdb-4dcd-aa9a-f27f8e91562a)

The latter allows the type inference to work in the component, is it possible to return the headers with this method, or do i have to use the former in order to appropriately return the cache control headers?

Reply:
Type inference for a loader that uses the `json()` helper works here: https://github.com/nasa-gcn/gcn.nasa.gov/blob/main/app/routes/circulars.%24circularId.tsx

Reply:
I figured it out, I was importing json from react-router, the example you provided was importing it from `'@remix-run/node'`. Should be all set in the next commit